### DISCUSSION TITLE
UEFI capsule updates not available or enabled in firmware setup

### DISCUSSION THREAD

**Discussion by solskogen** [fwupd/fwupd#5903] [Q&A]
I get this on a my MSI  MAG X570 TOMAHAWK WIFI (MS-7C84) and looking at dmesg it does not seem like there's anything related to ESRT at all. Does this mean that I can just forget it? Or is there some strange setting in the BIOS I can enable? I've tried searching for it, but can't find anything related to "allow Windows UEFI updates" or anything that's similar.

**Comment by hughsie** [comment]
> Does this mean that I can just forget it

Depends if you want capsule update support or not :) Maybe ask MSI if capsule updates are supported on that hardware?

**Reply by solskogen** [reply]
Heh, I guess I do. Don't I? Perhaps I don't quite understand what they are. I can try sending them an email.

**Reply by hughsie** [reply]
I think sending an email is a really great start. Let me know if you get anywhere. Thanks!
